static/coffee/services.coffee

```diff
@@ -251,12 +251,14 @@ services.factory('CaseService', ['$http', '$httpParamSerializer', '$window', ($h
     #----------------------------------------------------------------------------
     # Opens a new case
     #----------------------------------------------------------------------------
-    open: (message, summary, assignee, user) ->
-      params = {message: message.id, summary: summary}
+    open: (message, summary, assignee, user, urn) ->
+      params = {summary: summary, urn: urn}
```

Comment:
No this is still optional, so when it's not supplied when you open a case from a message, we will just send `{"urn": null}` in the JSON.